.github/workflows/publish-sdk.yaml

```diff
@@ -7,9 +7,11 @@ env:
   ANDROID_API_LEVEL: 33
 
 jobs:
-  lint-test-sdk:
+  test:
+    uses: ./.github/workflows/test.yaml
```

Comment:
Calls our test workflow ([GitHub docs](https://docs.github.com/en/actions/using-jobs/using-jobs-in-a-workflow#defining-prerequisite-jobs))